pkg/apis/klum.cattle.io/v1alpha1/types.go

```diff
@@ -21,10 +21,22 @@ type User struct {
 	Status            UserStatus `json:"status,omitempty"`
 }
 
+type GithubSyncSpec struct {
+	Owner       string `json:"owner"`
+	Repository  string `json:"repository"`
+	Environment string `json:"environment,omitempty"`
+	SecretName  string `json:"secret_name"`
```

Comment:
Inconsistent naming. Should be camelCased

```suggestion
	SecretName  string `json:"secretName"`
```